db/structure.sql

```diff
@@ -2913,3 +2913,5 @@ INSERT INTO "schema_migrations" (version) VALUES
 ('20200221220825'),
 ('20200303224134'),
 ('20200325201726');
+
+
```

Comment:
Nitpick but why is this added?

Reply:
No idea. I generated a new local dev db and seeded it. Not sure why Rails decided to recreate this artifact with 2 new blank lines. 🤷‍♂